packages/admin/admin/src/form/FinalFormInput.tsx

```diff
@@ -1,28 +1,44 @@
-import { InputBase, InputBaseProps } from "@mui/material";
+import { Button, InputBase, InputBaseProps } from "@mui/material";
 import * as React from "react";
 import { FieldRenderProps } from "react-final-form";
+import { FormattedMessage } from "react-intl";
 
 import { ClearInputAdornment } from "../common/ClearInputAdornment";
+import { useContentTranslationService } from "../translator/useContentTranslationService";
 
 export type FinalFormInputProps = InputBaseProps &
     FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement> & {
         clearable?: boolean;
+        disableContentTranslation?: boolean;
     };
 
-export function FinalFormInput({ meta, input, innerRef, endAdornment, clearable, ...props }: FinalFormInputProps): React.ReactElement {
+export function FinalFormInput({
+    meta,
+    input,
+    innerRef,
+    endAdornment,
+    clearable,
+    disableContentTranslation,
+    ...props
+}: FinalFormInputProps): React.ReactElement {
+    const { enabled, translate } = useContentTranslationService();
+
     return (
         <InputBase
             {...input}
             {...props}
             endAdornment={
-                clearable ? (
-                    <>
+                <>
+                    {clearable && (
                         <ClearInputAdornment position="end" hasClearableContent={Boolean(input.value)} onClick={() => input.onChange("")} />
-                        {endAdornment}
-                    </>
-                ) : (
-                    endAdornment
-                )
+                    )}
+                    {enabled && !disableContentTranslation && (
+                        <Button onClick={async () => input.onChange(await translate(input.value))}>
+                            <FormattedMessage id="comet.translate" defaultMessage="Translate" />
```

Comment:
We now have a `Translate` icon. Please use this icon together with a `Tooltip` instead of the plain message